packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -562,15 +550,10 @@ const PayableDetailsFormBase = forwardRef<
                                     InputLabelProps={{
                                       style: {
                                         color: isFieldRequired('tags')
-                                          ? 'red'
```

Comment:
Gosh! 😵

Reply:
Well, The PR was in draft

Reply:
Nope, the comment is for the **deleted** line. Hardcoded `red`... 🤮

Reply:
Ahh, true 😂